File: cmd/analyze/code-samples/split/js/sample-SPLIT.js
FeatureFlag.start("your_env_id", "your_api_key");

const visitor = FeatureFlag.newVisitor({
    visitorId: "your_visitor_id",
    context: { key: "value" },
});

visitor.on("ready", (error) => {
    if (error) {
        return;
    }

    const btnColorFlag = client.getTreatment(key, 'SPLIT-flag-js');

});

visitor.fetchFlags();

File: cmd/analyze/code-samples/split/java/sample-SPLIT.kt
val visitor1 = FeatureFlag.newVisitor("visitor_1")
            .context(hashMapOf("age" to "32", "isVIP" to true))
            .hasConsented(true)
            .isAuthenticated(true)
            .build()

val visitor1 = FeatureFlag.newVisitor("visitor_1").build()
visitor1.updateContext("isVip", true)
visitor1.fetchFlags().invokeOnCompletion {

    val btnColorFlag = visitor1.getTreatment(Key, "SPLIT-flag-kt");
}
